pkg/controller/nmstate/nmstate_controller.go

```diff
@@ -145,11 +147,28 @@ func (r *ReconcileNMState) applyRBAC(instance *nmstatev1beta1.NMState) error {
 
 func (r *ReconcileNMState) applyHandler(instance *nmstatev1beta1.NMState) error {
 	data := render.MakeRenderData()
+
+	// Prepare defaults
+	masterExistsNoScheduleToleration := corev1.Toleration{
+		Key:      "node-role.kubernetes.io/master",
+		Operator: corev1.TolerationOpExists,
+		Value:    string(corev1.TaintEffectNoSchedule),
+	}
+	webhookNodeSelector := map[string]string{
+		"beta.kubernetes.io/arch":        "amd64",
+		"node-role.kubernetes.io/master": "",
```

Comment:
name should be what it is `amd64ArchOnMasterNodeSelector`

Reply:
Fixed